index.js

```diff
@@ -26,94 +27,52 @@ class CumulusMessageAdapterExecutionError extends Error {
 async function generateCMASpawnArguments(command) {
   const adapterDir = process.env.CUMULUS_MESSAGE_ADAPTER_DIR || './cumulus-message-adapter';
   const systemPython = await lookpath('python');
-  if (systemPython) {
+  if (systemPython && process.env.USE_CMA_BINARY !== 'true') {
     return [systemPython, [`${adapterDir}`, command]];
   }
   // If there is no system python, attempt use of pre-packaged CMA binary
-  return [`${adapterDir}/cma`, [command]];
+  return [`${adapterDir}/cma_bin/cma`, [command]];
 }
 
+
 /**
  * Invoke the cumulus-message-adapter
  *
- * @param {string} command - the action to be performed by the message-adapter
- * @param {Object} input - the input to be sent to the message-adapter
  * @returns {Promise.<Object>} - the output of the message-adapter
  */
-async function callCumulusMessageAdapter(command, input) {
-  const spawnArguments = await generateCMASpawnArguments(command);
+async function invokeCumulusMessageAdapter() {
+  const spawnArguments = await generateCMASpawnArguments('stream');
+  const errorObj = { stderr: '' };
```

Comment:
I might prefer to call this the `stdErrBuffer` or something, but maybe that's a misnomer

Reply:
9a15062